ebmlite/core.py

```diff
@@ -35,36 +35,34 @@
     Eventually, recognize official schemata when loading, like the system
     currently handles legacy ``python-ebml`` schemata.
 '''
-from __future__ import division, absolute_import, print_function, unicode_literals
+from __future__ import absolute_import, division, print_function, unicode_literals
 
-__author__ = "David Randall Stokes"
-__copyright__ = "Copyright 2018 Mide Technology Corporation"
+from future import standard_library
+standard_library.install_aliases()
+from builtins import bytes
+from past.builtins import basestring
+from builtins import object
+__author__ = b"David Randall Stokes"
```

Comment:
Metadata is supposed to go after `__future__` imports but before regular imports (PEP8).